rwatch/ui/test/graphics_standalone_tests.c

```diff
@@ -23,23 +23,23 @@ void main(void)
 
 void test_window_layer(void)
 {
-    printf("testing Window/Layers\n");
+	SYS_LOG("graphics_test", APP_LOG_LEVEL_INFO, "testing Window/Layers");
```

Comment:
Check indentation (this seems to convert four spaces into a hard tab)

Reply:
Whoops